---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

Education
======
* B.S. in Computer Science, Stony Brook University, 2006
* Ph.D in Applied Statistics, Stony Brook University, 2011

Work experience
======
* 2019 - Current: Assistant Professor
  * Department of Statistics, Miami University

* 2015 - 2019: Computational Scientist
  * Computational Sciences, Jackson Laboratory

* 2013 - 2015: NIH/NIDA Research Fellow
  * Department of Psychiatry, Medical College of Virgina
  * Virginia Commonwealth University

* 2011 - 2013: Research Fellow
  * Department of Psychiatry, Medical College of Virgina
  * Virginia Commonwealth University

Publications
======
* Peer Reviewed Publication
  * Lawlor N, Marguez EJ, __Lee D*__, Ucar D. (2020). V-SVA: an R shiny application for detecting and annotating sources of variation in single cell RNA-seq data. To appear in Bioinformatics.
  * __Lee D*__, Cheng A, Lawlor N, Bolisetty M, Ucar D. (2018). Detection of correlated hidden factors from single cell transcriptomes using Iteratively Adjusted-SVA (IA-SVA), Scientific Reports 8, Article number:17040.
  * Chatzinakos C, __Lee D__, Webb BT, Vladimirov VI, Kendler KS, Bacanu SA. (2017). JEPEGMIX2: improved gene-level joint analysis of eQTLs in cosmopolitan cohorts, Bioinformatics, doi.org/10.1093/bioinformatics/btx509.
  * Bigdeli TB, __Lee D__, Riley BP, Vladimirov VI, Fanous AH, Kendler KS and Bacanu SA. (2016). A simple yet accurate correction for winner’s curse can predict signals discovered in much larger genome scans, Bioinformatics, doi: 10.1093/bioinformatics/btw303.
  * __Lee D*__, Williamson VS, Bigdeli TB, Riley BP, Webb BT, Fanous AH, Kendler KS, Vladimirov VI, Bacanu SA. (2015). JEPEGMIX: gene-level joint analysis of functional SNPs in cosmopolitan cohorts, Bioinformatics, doi: 10.1093/bioinformatics/btv567.
  * Edwards AC, Bigdeli TB, Docherty AR, Bacanu SA, __Lee D__, de Candia TR, Moscati A, Thiselton DL, Maher BS, Wormley BK, Molecular Genetics of Schizophrenia Collaboration (MGS), Walsh D, O'Neill FA, Kendler KS, Riley BP, Fanous AH. (2015). Meta-analysis of positive and negative symptoms reveals schizophrenia modifier, Schizophrenia Bulletin, doi: 10.1093/schbul/sbv119.
  * __Lee D*__, Bigdeli TB, Williamson VS, Vladimirov VI, Riley BP, Fanous AH, Bacanu SA. (2015). DISTMIX: Direct imputation of summary statistics for unmeasured SNPs from mixed ethnicity cohorts, Bioinformatics, doi: 10.1093/bioinformatics/btv348.
  * Williamson VS, Mamdani M, McMichael GO, Kim AH, __Lee D__, Bacanu SA, Vladimirov VI. (2015). Expression Quantitative Trait Loci (eQTLs) in microRNA Genes are Enriched for Schizophrenia and Bipolar Disorder Association Signals, Psychological Medicine, doi: 10.1017/S0033291715000483.
  * Docherty AR, Bigdeli TB, Edwards AC, Bacanu SA, __Lee D__, Neale MC, Wormley BK, Walsh D, O’Neill FA, Riley BP, Kendler KS, Fanous AH. (2015). Gene-based and gene pathway analysis of symptom dimensions of psychotic illness based on schizophrenia-specific model of the OPCRIT, Schizophrenia Research, doi: 10.1016/j.schres.2015.02.013.
  * __Lee D*__, Williamson VS, Bigdeli TB, Riley BP, Fanous AH, Vladimirov VI, Bacanu SA. (2014). JEPEG: a summary statistics based tool for gene-level joint testing of functional variants, Bioinformatics, doi: 10.1093/bioinformatics/btu816.
  * __Lee D*__, Bacanu SA. (2013). Association testing strategy for data from dense marker panels, PloS One, doi: 10.1371/journal.pone.0080540.
  * __Lee D*__, Bigdeli TB, Riley BP, Fanous AH, Bacanu SA. (2013). DIST: direct imputation of summary statistics for unmeasured SNPs, Bioinformatics doi: 10.1093/bioinformatics/btt500. <br/>
    __`*`__ Corresponding Author  

* Non-peer reviewed preprints
  * Chatzinakos C, __Lee D__, Na C, Docherty A, Vladimirov VI, Webb BT, Riley BP, Flint J, Kendler KS,
 Bacanu SA. (2018). JEPEGMIX2-P: Novel pathway transcriptomic method greatly increases detection of
 molecular pathways in cosmopolitan cohorts, bioRxiv, doi: https://doi.org/10.1101/373050.
  * __Lee D__, Bigdeli TB, Vladimirov VI, Fanous AH, Bacanu SA. (2016). QCAT: testing causality of variants using only summary association statistics, bioRxiv, doi: https://doi.org/10.1101/072355.



<!--
  <ul>{% for post in site.publications %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
-->

Teaching
======
* STA 463/563 Regression Analysis, Miami University
* STA 402/502 Statistical Programming, Miami University
* MEDS 6498 Topics in Bioinformatics and Computational Biology, University of Connecticut Health Center
* CCTR 702 Statistics for Genetic Studies I, Virginia Commonwealth University
* HPH 507 Biostatistics II, Stony Brook University
* AMS 110 Probability and Statistics, Stony Brook University

<!--
  <ul>{% for post in site.teaching %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
-->

<!--
Talks
======
  <ul>{% for post in site.talks %}
    {% include archive-single-talk-cv.html %}
  {% endfor %}</ul>
-->

<!--
Service and leadership
======
* 
-->
